services/graph/pkg/service/v0/educationclasses.go

```diff
@@ -362,9 +362,9 @@ func (g Graph) PostEducationClassMember(w http.ResponseWriter, r *http.Request)
 // DeleteEducationClassMember implements the Service interface.
 func (g Graph) DeleteEducationClassMember(w http.ResponseWriter, r *http.Request) {
 	logger := g.logger.SubloggerWithRequestID(r.Context())
-	logger.Info().Msg("calling delete class member")
 
 	classID := chi.URLParam(r, "classID")
+	logger.Info().Str("classID", classID).Msg("calling delete class member")
```

Comment:
we should also log the deleted member

Reply:
Did also with the other `DELETE` endpoints